run

```diff
@@ -246,10 +250,18 @@ if ! find "$resources" -name '*.java' | xargs javac -cp ".:${resources}:${workli
         'staff'
 fi
 
+# Find all .properties-files in the resources folder and copy them here
```

Comment:
Isn't it possible to just add the properties directory to the class path?

Reply:
I have tried to do so, but the Java judge does not seem to be able to find the file.